format-common/src/main/java/io/cdap/plugin/format/input/PathTrackingInputFormatProvider.java

```diff
@@ -63,6 +65,18 @@ protected void validate() {
     // no-op
   }
 
+  protected Schema getSchema(Schema dataSchema) {
```

Comment:
getSchema is a confusing name, rename it to addPathField.

Reply:
Changed